docs/TROUBLE.md

```diff
@@ -0,0 +1,162 @@
+
+- If you can't change flight modes or download parameters or issue other commands via the ground station, check your wiring to ensure that the correct serial tx output on your ground control device or wireless module or serial adapter is connected to the correct serial rx input on your Tx module.
+
+- If flight mode changes initiated by the ground control system are correctly reported back to the ground control system and FM sensor, but parameter download fails, see [Confirm reliable message delivery](TROUBLE.md#Confirm reliable message delivery) below.
+
+## 8. Confirm RC control is passed from the radio to the flight controller
+
+Use the radio calibration page of your ground station (connected either via your Tx module or directly to USB) to verify all RC controls are working as expected.
+
+- If you use RC/_CHANNELS\_OVERRIDE instead of a CRSF Rx output and are testing with QGroundControl, switch to Mission Planner or APM Planner 2.0 since QGroundControl doesn't show RC\_CHANNELS\_OVERRIDE on this page.
+
+- If the display doesn't follow stick movement and you use a CRSF connection between your Rx and your flight controller, check that you have connected the wire to the correct pins/pads.
+
+- For CRSF, check that you have correctly [configured ArduPilot](ARDUPILOT.md#crsf-receiver) to use this input.
+
+- For CRSF, check that you have correctly [configured your Rx](CRSF.md#mlrs-rx-module-setup) for crsf output.
+
+- If your Rx [doesn't provide an RC output](ELRS_RECEIVERS.md#connections) or you don't want to use it, check that you have set [Rx Snd RcChannel](PARAMETERS.md#rx-snd-rcchannel) to rc override.
+
+## 9. Confirm reliable message delivery
+
+Use your ground control station connected to your Tx module to examine the message loss rate both during parameter download and for a few minutes after it completes.
+
+For QGroundControl, select Application Settings -> MAVLink and scroll down to MAVLink Link Status.
+
+For Mission Planner, select status in the upper right, move the windows so you can see both and then refresh parameters from the full parameters page.
+
+You should see very few or no messages lost.
+
+- If you see more than the occasional lost message or if parameter download takes more time than expected (8 to 60 seconds, depending on mode and ArduPilot version), check that [rx-ser-link-mode](PARAMETERS.md#rx-ser-link-mode) is set to "mavlink" or "mavlinkX" and that [rx-snd-radiostat](PARAMETERS.md#rx-snd-radiostat) is set to "ardu_1" or "meth_b"
+
+- Check your [ArduPilot setup](CRSF.md#ardupilot-setup) to be sure that the baud rate between the Rx and  and your configured Stream Rates are appropriate for your radio mode.
+
+- Upgrade to a newer version of ArduPilot which may have fixed flow control issues.
+
+## Range issues
+
+If you don't see the range you expected, consider the following.
+
+- Never power up a Rx or Tx module without an antenna connected as it may cause damage, especially at high output levels.
```

Comment:
this is actually not correct as such, e.g. the MatekSys will hanlde it,.
So maybe rather along the lines "Some Rx and Tx modules can be destroyed when powered up without an antenna, so it is highly recommended to ...."

Reply:
That's why I said "may"  Some designs can tolerate it, but in my opinion, it's not a habit you want to start, even if a manufacturer says it's OK.

Reply:
yes ... but IMHO it's still a very different message which is transported

Reply:
I've reworded.  Hopefully better now.